spec/basic/expected_conditions_spec.js

```diff
@@ -66,6 +67,18 @@ describe('expected conditions', function() {
     expect(EC.titleIs('My AngularJS App').call()).toBe(true);
   });
 
+  it('should have urlContains', function() {
+      var baseUrlFromSpec = browser.baseUrl;
+      expect(EC.urlContains('/form').call()).toBe(true);
```

Comment:
@cnishina Just realized what you meant. Fixed the hardcoded url with env.baseUrl